lib/tasks/demo.rake

```diff
@@ -158,7 +215,8 @@ desc 'Backup the database'
     cache_analytics = (ENV['SKIP_ANALYTICS'] != 'true')
 
     jurisdictions = Jurisdiction.all
-    assigned_users = Hash[jurisdictions.pluck(:id).map {|id| [id, (1..999_999).to_a.sample(10)]}]
+    assigned_users_range = (1..9_999).to_a.freeze
+    assigned_users = Hash[jurisdictions.pluck(:id).map {|id| [id, assigned_users_range]}]
```

Comment:
@Bialogs I think mentioned that in prod there are sometimes thousands of assigned users so for performance testing we wanted to have a large array. I also figured that storing one of these arrays an memory and referencing it in the next line would save memory since it just needs to point to `assigned_users` and save CPU time in that we aren't generating random numbers for every assigned users array

Reply:
@jkufro not sure if I understand what you mean, would we be able to get around storing a list of assigned users for every jurisdiction?

Reply:
The issue is only having 10 assigned users per jurisdiction. We have some jurisdictions that are using 10,000 unique assigned user values.

Reply:
can't we accomplish that by writing:
```
assigned_users = Hash[jurisdictions.pluck(:id).map { |id| [id, 10_000.times.map { |n| Faker::Number.number(digits: 6) }] }]
```
or even having a variable number for number of different jurisdictions?
```
assigned_users = Hash[jurisdictions.pluck(:id).map { |id| [id, rand(1..10000).times.map { |n| Faker::Number.number(digits: 6) }] }]
```

Reply:
@timwongj The method you suggested is much slower. Keeping a static array seems to be the way to go in terms of performance.